test/svm/SvmSpoke.SlowFill.ts

```diff
@@ -381,7 +381,9 @@ describe("svm_spoke.slow_fill", () => {
       )
       .accounts(executeSlowRelayLeafAccounts)
       .remainingAccounts(fillRemainingAccounts)
-      .rpc();
+      .instruction();
+    const computeBudgetInstruction = ComputeBudgetProgram.setComputeUnitLimit({ units: 1_400_000 });
```

Comment:
Has the borsh serialisation increased the CU?

Reply:
Its hard to tell precisely as our tests are not deterministic and have much more variance due to differing bumps is PDA derivation, but over 10 runs I observed on average +2% in CU, which I guess is acceptable here.